marlon/baseline_models/ppo/train_defender.py

```diff
@@ -5,7 +5,7 @@
 from marlon.baseline_models.multiagent.random_marlon_agent import RandomAgentBuilder
 
 ENV_MAX_TIMESTEPS = 2000
-LEARN_TIMESTEPS = 10_000
+LEARN_TIMESTEPS = 10_0000
```

Comment:
Rip @Chrisnguyen3 's PC haha

Reply:
Lol yeah. I upped because I was hoping to see improvement as it runs. (Depending on how things go we can tune how long the training will actually take)